src/test/java/emissary/test/core/junit5/RegressionTestUtil.java

```diff
@@ -118,7 +118,7 @@ public static Path getTestResx() {
     }
 
     /**
-     * @see ExtractionTest#checkAnswers(Document, IBaseDataObject, List, String)
+     * See {@link ExtractionTest#checkAnswers(Element, IBaseDataObject, List, String)}
```

Comment:
```suggestion
     * See {@link ExtractionTest#checkAnswers(Document, IBaseDataObject, List, String)}
```
This reference is still valid, right?

Reply:
Good pickup here actually. The method with the original signature does exist, I accidentally had it pointing to the other checkAnswers method. I'll switch it to reference the right one.